playground/src/fly-in/base.js

```diff
@@ -41,20 +47,12 @@ class FlyIn {
   registerButtons(target) {
     const button = target.querySelector('.fly-in-close');
     button.addEventListener('click', () => {
-      this.hideFlyIn(target);
+      this.toggleFlyIn(false);
     });
   }
 
-  toggle() {
-    this.target.classList.toggle('active');
-  }
-
-  hideFlyIn() {
-    this.target.classList.remove('active');
-  }
-
-  showFlyIn() {
-    this.target.classList.add('active');
+  toggleFlyIn(force) {
```

Comment:
Why not simply `toggle` to get rid of the duplicate wording? The same applies to `createFlyIn`.